src/isomorphic/classic/element/__tests__/ReactElementValidator-test.js

```diff
@@ -95,9 +99,10 @@ describe('ReactElementValidator', function() {
     ReactTestUtils.renderIntoDocument(<Anonymous>{divs}</Anonymous>);
 
     expect(console.error.argsForCall.length).toBe(1);
-    expect(console.error.argsForCall[0][0]).toBe(
+    expect(normalizeCodeLocInfo(console.error.argsForCall[0][0])).toBe(
       'Warning: Each child in an array or iterator should have a unique ' +
-      '"key" prop. See https://fb.me/react-warning-keys for more information.'
+      '"key" prop. See https://fb.me/react-warning-keys for more information.\n' +
+      '    in div (at **)'
```

Comment:
It would be nice if this had both the parent component (Anonymous, or "Unknown" as we'll know it as in the devtools) and the child (div) here. Is that possible? I guess we would probably have to make getCurrentStackAddendum take both elements as arguments.